warehouse/templates/pages/help.html

```diff
@@ -36,23 +65,57 @@ <h3 id="packages" class="common-question">What's a package, project, or release?
 
       <p>A <em>file</em>, also known as a <em>package</em>, on PyPI is something that you can download and install. Because of different hardware, operating systems, and file formats, a release may have several files (packages), like an archive containing source code or a binary <a href="https://pypi.org/project/wheel/#files">wheel</a>.</p>
 
-
-      <h3 id="installing" class="common-question">How do I install a file (package) from PyPI?</h3>
+      <h3 id="installing" class="common-question">{{ installing() }}</h3>
 
       <p>To learn how to install a file from PyPI, visit the <a href="https://packaging.python.org/installing/">installation tutorial</a> on the <a href="https://packaging.python.org/">Python Packaging User Guide</a>.</p>
 
-
-      <h3 id="publishing" class="common-question">How do I package and publish my code for PyPI?</h3>
+      <h3 id="publishing" class="common-question">{{ publishing() }}</h3>
 
       <p>For full instructions on configuring, packaging and distributing your Python project, refer to the <a href="https://packaging.python.org/distributing/">packaging tutorial</a> on the <a href="https://packaging.python.org">Python Packaging User Guide</a>.</p>
 
+      <h3 id="uploading" class="common-question">{{ uploading() }}</h3>
+
+      <p>This feature was deprecated with the new version of PyPI - we instead recommend that you <a href="https://packaging.python.org/tutorials/distributing-packages/#uploading-your-project-to-pypi">use twine to upload your project to PyPI</a>.</p>
+
+      <h3 id="preproduction" class="common-question">{{ preproduction() }}</h3>
+
+      <p>'Pre-production' indicates that you are using a version of this software that is not fully tested and 'production ready'. This means:
+      <ul>
+        <li>The UI may return unusual or erroneous results</li>
+        <li>Performance may not yet be optimized</li>
+        <li>Some features that you use on the old interface may be missing</li>
+      </ul>
+
+      <p>While we are in pre-production, our infrastructure cannot (yet!) support all of the traffic generated by our users running <code>pip install</code>. We therefore ask that you do not use this version of PyPI for production installations.</p>
+      <p>On the other hand, uploads are working well and we encourage project maintainers and owners to use this version of PyPI over the legacy site.</p>
```

Comment:
One slight tweak here, not sure if it matters but more than encouraging people to upload using Warehouse, we *require* it. Uploading to legacy no longer functions. Not sure if it makes sense to change this wording or not.

Reply:
Proposed new text:

`On the other hand, uploads are working well and project maintainers and owners should use this version of PyPI over the legacy site, which no longer supports this function.`

is this ok @dstufft ?

Reply:
Yup, sounds good.